portlets/src/main/webapp/vue-app/engagement-center/components/common/DescriptionEditor.vue

```diff
@@ -135,12 +135,12 @@ export default {
       this.editor = CKEDITOR.instances['descriptionContent'];
       $(this.$refs.editor).ckeditor({
         customConfig: '/commons-extension/ckeditorCustom/config.js',
-        extraPlugins: 'simpleLink,widget',
+        extraPlugins: 'simpleLink,widget,emoji',
```

Comment:
Please follow the same implementation of the plugin as https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue#L201-L212